engine-core/src/test/java/io/nosqlbench/engine/core/ActivityExecutorTest.java

```diff
@@ -65,7 +65,7 @@ public synchronized void testRestart() {
         try {
             ad.setThreads(1);
             ae.startActivity();
-            ae.stopActivity();
+            ae.stopActivity(false);
```

Comment:
I'm not a huge fan of using booleans to encode critical details like this. It's not very self explanatory.
Can we break this out into an `ae.forceStopActivity()` for example?

Reply:
I will make the changes.

Reply:
@jshook change have been made. A separate forceStopActivity() method is added.